mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1141,13 +1141,24 @@ def _get_args_for_metrics(self, extra_metric, eval_df):
                     ):
                         eval_fn_args.append(self.other_output_columns[column])
                     elif param.default == inspect.Parameter.empty:
+                        output_column_name = self.evaluator_config.get(
+                            _Y_PREDICTED_OUTPUT_COLUMN_NAME, "output"
+                        )
+                        output_columns = [output_column_name]
+                        if self.other_output_columns is not None:
+                            output_columns.extend(self.other_output_columns.columns)
+                        input_columns = list(input_df.columns)
```

Comment:
I felt like it would be confusing if the output columns is shown as None/empty or the output_column_name is excluded - but I guess it could also be confusing if it is shown (since we cannot actually map to that column).

Reply:
updated!